src/components/ChatApp/MessageSection/MessageSection.react.js

```diff
@@ -864,11 +864,12 @@ class MessageSection extends Component {
     let indexLimit = this.state.searchState.scrollLimit;
     let markedIDs = this.state.searchState.markedIDs;
     let ul = this.messageList;
-    if (newSearchCount <= 0) {
-      newSearchCount = indexLimit;
+    if (newSearchCount > indexLimit) {
```

Comment:
This file should not have been changed according to your PR description. Plus there is already a PR regarding this file.

Reply:
@AakashMallik I am a bit confused here. I opened a new branch for new pull request. But the commits got merged into previous one!Can you please help me out here.

Reply:
Before opening a new branch, always go back to master first. Probably the reason the commits from the other branch ended up here is because to branched out from the wrong branch. I hope that makes it clear.

Reply:
@sunitasen Did you end up merging the changes into master? That would explain this. But to get out of this situation, I would advise you to delete your master, create a master again branched from upstream/master, them branch out from master into a new branch and then make this changes. And before doing this, save the files you have made changes to somewhere on your HDD.

Reply:
@AakashMallik  it worked I think. Thanks a ton :smile: